### PR TITLE
[12.0][IMP] POS_multiplecontrol : Automatic_solve name

### PR DESCRIPTION
**Pull request by quentinDupont**
Add journal name in automaticsolve statement
Improve french translation

@legalsylvain

### CODE REVIEW

**Review by legalsylvain — APPROVED** [review]
Thanks !

Note : not sure the translation part will be taken into accout. I think that for existing module, translation will be erased by weblate.

Not a blocking point.

**Comment by quentinDupont** [comment]
> Thanks !
> 
> Note : not sure the translation part will be taken into accout. I think that for existing module, translation will be erased by weblate.
> 
> Not a blocking point.

Right ! Well i will do the translation in weblate when it will be merged

File: pos_multiple_control/models/account_bank_statement.py [hunk 1]
```diff
@@ -114,7 +115,8 @@ def automatic_solve(self):
                         'journal_id': cb_journal_id,
                         'statement_id': statement.id,
                         'amount': abs(cb_difference),
-                        'name': 'Automatic solve',
+                        'name': _('Automatic solve') + " ("
+                                                     + cb_journal_name + ")",
```
Comment (ivantodorovich) [inline pos_multiple_control/models/account_bank_statement.py]:
```suggestion
                        'name': _('Automatic solve (%s)') % cb_journal_name,
```

**Review by ivantodorovich — APPROVED** [review]
small suggestion

**Comment by legalsylvain** [comment]
Hi @ivantodorovich thanks for your review and your suggestion.

/ocabot merge patch

**Comment by legalsylvain** [comment]
hi @quentinDupont. check travis log. https://travis-ci.com/github/OCA/pos/jobs/434733711#L2953

that's weird, but it seems po file is not good. maybe reinstall the module from scratch and generate again the PO file will fix the bug...

**Pull request merged**